### DISCUSSION TITLE
Specify Algorand Manager Address when calling stdlib.launchToken

### DISCUSSION THREAD

**Discussion by lorenzopolidori** [reach-sh/reach-lang#1449] [Q&A] [answered]
In the [Reach docs](https://docs.reach.sh/frontend/#p_178), the Manager Address is not part of the list of Algorand-specific parameters that can be passed to `stdlib.launchToken`.

However, `ManagerAddr` is one of the optional Algorand `AssetParams`.

Can we add support for a `manager` parameter in `stdlib.launchToken`?

**Comment by jeapostrophe** [comment, marked as answer]
You can track this issue for that --- https://github.com/reach-sh/reach-lang/issues/1440 --- it will be added soon
